core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala

```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
     <table class="table table-bordered table-striped table-condensed sortable table-fixed">
       <thead>
         <th>Pool Name</th>
-        <th>Minimum Share</th>
-        <th>Pool Weight</th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pools can be set a minimum share
```

Comment:
These are a bit too long for tooltips. They are meant to elaborate, very briefly, the immediate meaning of the column. More like "Pool's minimum share of CPU cores"

Reply:
"Pool's minimum share of CPU cores" - This is fine, i will updated the code